.github/workflows/backport.yml

```diff
@@ -0,0 +1,24 @@
+name: Backport
+on:
+  pull_request_target:
+    types:
+      - closed
+      - labeled
+jobs:
+  backport:
+    name: Backport
+    runs-on: ubuntu-latest
+    if: >
+      github.event.pull_request.merged
+      && (
+        github.event.action == 'closed'
+        || (
+          github.event.action == 'labeled'
+          && contains(github.event.label.name, 'backport-to-')
```

Comment:
Nitpicking: I think that `startWith(github.event.label.name, 'backport-to-')` would be better here.

WDYT?

Reply:
Sure, why not. Done.